### PR TITLE
Adding Support for int64 

### PR DESCRIPTION
**Pull request by medyagh**
also added glide.yaml for dependency management. solve this issue https://github.com/caarlos0/env/issues/23

### CODE REVIEW

File: env_test.go [hunk 1]
```diff
@@ -5,8 +5,7 @@ import (
 	"os"
 	"testing"
 	"time"
-
-	"github.com/caarlos0/env"
+	"env"
```
Comment (caarlos0) [inline env_test.go]: I think you'll have to revert this..

Comment (medyagh) [inline env_test.go]: ok I will try :)

**Comment by medyagh** [comment]
yay ! all checks passed. ready to be merged please :) ?

**Comment by caarlos0** [comment]
Done, thank you very much!

**Comment by caarlos0** [comment]
closes #23

**Pull request merged**